### PR TITLE
OCPBUGS-17499: Add iDRAC9 prereq for ISO boot via Redfish API

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.13, 4.14., 4.15

Issue: https://issues.redhat.com/browse/OCPBUGS-17499

Link to docs preview: https://70958--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_sno/install-sno-installing-sno#install-booting-from-an-iso-over-http-redfish_install-sno-installing-sno-with-the-assisted-installer

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/install-booting-from-an-iso-over-http-redfish.adoc [hunk 1]
```diff
@@ -21,6 +21,7 @@ Ensure that you have the latest firmware version of iDRAC that is compatible wit
 .Prerequisites
 
 * Download the installation {op-system-first} ISO.
+* Use iDRAC 9 based Dell PowerEdge server.
```
Comment (dfitzmau) [inline modules/install-booting-from-an-iso-over-http-redfish.adoc:24]: "based" is placed here among a string of adjectives, so insetad of hyphenating the term to `-iDRAC-9-based Dell PowerEdge server`, how about `Use an iDRAC 9 X that is compatible with a Dell PowerEdge server`. WDYT?

Comment (amolnar-rh) [reply modules/install-booting-from-an-iso-over-http-redfish.adoc:24]: Changed to
> Use a Dell PowerEdge server that is compatible with iDRAC9.

Comment (dfitzmau) [reply modules/install-booting-from-an-iso-over-http-redfish.adoc]: Yeah, your update is much better. LGTM!

**Pull request merged**